src/include/detail/flat/vq.h

```diff
@@ -223,7 +223,7 @@ auto vq_query_heap_tiled(
   } while (load(db));
 
   consolidate_scores(scores);
-  auto top_k = get_top_k_with_scores(scores, k_nn);
+  auto top_k = get_top_k_with_scores<fixed_min_pair_heap<float, Index>, Index>(scores, k_nn);
```

Comment:
The `get_top_k` functions deduce the types from the arguments (or they do in my current branch).  I'll make sure all of those functions are consistent one way or the other and propagate those changes to this PR.